src/python/WMCore/Storage/StageOutImpl.py

```diff
@@ -147,8 +176,16 @@ def createDebuggingCommand(self, sourcePFN, targetPFN, options=None, checksums=N
         Build a shell command that will report in the logs the details about
         failing stageOut commands
         """
-        raise NotImplementedError("StageOutImpl.createDebuggingCommand")
```

Comment:
I think we should keep this NotImplementedError exception, because we want to force each stage out plugin to actually override this implementation.

Was there any strong reason that you wanted to have a "dummy cp" thingy here?

Reply:
On second thought, I agree with you that it's better to keep the `NotImplementedError` in the parent class. I will roll it back